### PR TITLE
Grid Toast and Text Utils

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have added Grid, Toast, and Text Utils for my Component Library.

Please review and suggest.

### CODE REVIEW

**Review by ShekharDhangar — APPROVED** [review]
Looks great, good that you are using helper classes instead of re-writing all properties

**Review by aayushjain-code — APPROVED** [review]
Everything looks good to me, try to add a bit of border-radius to grid objects

File: components/utils/color-utils.css [hunk 1]
```diff
@@ -5,7 +5,7 @@
   color: var(--white);
 }
 
-.secondary{
+.secondary {
   background-color: var(--secondary);
   color: var(--white);
```
Comment (MuskanChhatrasal) [inline components/utils/color-utils.css]: It's not recommended to use color name as variable name. 
Else LGTM!!

**Review by MuskanChhatrasal — APPROVED** [review]

File: components/utils/color-utils.css (hunk 1 above)
Comment (ApurvaSawant11) [reply components/utils/color-utils.css]: I have made the changes. Thanks

**Comment by ApurvaSawant11** [comment]
> Everything looks good to me, try to add a bit of border-radius to grid objects

Done. Thanks!

**Pull request merged**